chemprop/features/featurization.py

```diff
@@ -44,16 +50,28 @@ def set_extra_atom_fdim(extra):
     EXTRA_ATOM_FDIM = extra
 
 
-def get_bond_fdim(atom_messages: bool = False) -> int:
+def get_bond_fdim(atom_messages: bool = False,
+                  overwrite_default_bond: bool = False,
+                  overwrite_default_atom: bool = False) -> int:
     """
```

Comment:
I think that `get_bond_fdim` is called without the overwrite arguments as part of `get_components` when it needs them. (can't figure out how to drop a comment there since it is inactive code).

Reply:
Good catch - it should be in both functions (init and get_components, otherwise there will be another bug with `--atom_messages`) - if [PR#138](https://github.com/chemprop/chemprop/pull/138) is merged before this, you should also remove the previous bug we have with atom messages to avoid a merge conflict

Reply:
Nice catch!